public/app/features/alerting/unified/components/mute-timings/MuteTimingForm.tsx

```diff
@@ -63,11 +58,8 @@ const MuteTimingForm = ({ muteTiming, showError, loading, provenance }: Props) =
   const [updating, setUpdating] = useState(false);
 
   const defaultAmCortexConfig = { alertmanager_config: {}, template_files: {} };
-  const amConfigs = useUnifiedAlertingSelector((state) => state.amConfigs);
-  const { result = defaultAmCortexConfig } =
-    (selectedAlertmanager && amConfigs[selectedAlertmanager]) || initialAsyncRequestState;
+  const { result = defaultAmCortexConfig, config } = useAlertmanagerConfig(selectedAlertmanager);
```

Comment:
why do we need to set this default value here? is not the `useAlertmanagerConfig` doing the same?

Reply:
It does actually, I think we can safely remove this by just making sure we check for `if (result)` on the `onSubmit` handler.